src/scenes/Event/components/Sidebar.js

```diff
@@ -77,7 +84,7 @@ const Heading = styled(Typography).attrs({ variant: 'body1' })`
 const Content = styled(Typography).attrs({ variant: 'title' })`
```

Comment:
convert to JSS

Reply:
Why needed convert to JSS?

Reply:
We are preferring to switch any Styled Components to JSS.